ui/js/directives/treeherder/log_viewer.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+treeherder.directive('thLogViewer', ['$sce', '$location', ($sce, $location) => {
```

Comment:
Since we're here, we might want to switch to using an angular component instead of a directive, which is a bit simpler... there's a few examples in `ui/js/components/`